### DISCUSSION TITLE
Plasteel Surgery (Continued) - 번역 갱신

### DISCUSSION THREAD

**Discussion by TofuTamago** [RimWorldKorea/RMK#282] [일반]
[Plasteel Surgery (Continued) - 2018276375 - Korean.zip](https://github.com/RimWorldKorea/RMK/files/13684348/Plasteel.Surgery.Continued.-.2018276375.-.Korean.zip)

" Recipe_Surgery.xml " 파일에서 " 신체 유형 : 근육 성장 자극 " 옆에 있던 한자 삭제
" Messages.xml " 파일에서 미처 못 보고 번역을 안 했던 일본어 문구 번역 (성형수술 완료 시 나오는 문구)

**Comment by RimWorldKorea** [comment]
저번에 해당 모드 업데이트 하면서 xlsx 파일을 동봉해두었습니다.

해당 파일에 정리해서 올려줄 수 있을까요?

다른 파일 필요없이 xlsx 파일만 올려주시면 됩니다.
